projects/solarized/README.md

```diff
@@ -341,6 +345,11 @@ These screen shots show Vim running with my own [Pandoc Kit Syntax](http://ethan
 [![tex dark](https://github.com/altercation/solarized/raw/master/img/screen-tex-dark-th.png)](https://github.com/altercation/solarized/raw/master/img/screen-tex-dark.png)
 [![tex light](https://github.com/altercation/solarized/raw/master/img/screen-tex-light-th.png)](https://github.com/altercation/solarized/raw/master/img/screen-tex-light.png)
 
+### Atom
+
+[![atom dark](https://cloud.githubusercontent.com/assets/378023/12602908/9c252b08-c4f0-11e5-8833-6aad91b8fa34.png)](https://cloud.githubusercontent.com/assets/378023/12602908/9c252b08-c4f0-11e5-8833-6aad91b8fa34.png)
```

Comment:
I can thumbnail these if it's critical. I thought it was nice to show the full screenshot inline.